subsys/mgmt/mcumgr/lib/cmd/img_mgmt/Kconfig

```diff
@@ -84,6 +84,22 @@ config IMG_MGMT_FRUGAL_LIST
 	  a device but requires support in client software, which has to default omitted values.
 	  Works correctly with mcumgr-cli.
 
+config MGMT_MCUMGR_GRP_IMG_UPLOAD_CHECK_HOOK
```

Comment:
Sorry for asking to change this again, but I have agreed that the prefix `MGMT_` is too much, and `MCUMGR_` is distinctive enough.